src/ObjectMultiplex.ts

```diff
@@ -23,6 +23,11 @@ export class ObjectMultiplex extends Duplex {
   }
 
   createStream(name: string): Substream {
+    // validate stream destroyed already
+    if (this.destroyed) {
```

Comment:
Actually, should we check for `ended` as well? In case somebody tried to gracefully stop the stream.

Reply:
doesnt look like it has `ended`

![image](https://user-images.githubusercontent.com/364566/120227824-cb347a00-c1fe-11eb-92d9-55a89ac623fd.png)

Reply:
Probably a type issue. I think the `readable-stream` implementation we use (`2.3.3`, matching Node 8 streams) has the property: https://github.com/nodejs/readable-stream/blob/cd59995050105b946884ee20e3bcadc252feda8c/lib/_stream_readable.js#L131